tests/ac/mir/screencast_tests.cpp

```diff
@@ -167,8 +167,8 @@ TEST(Screencast, NoPixelFormatAvailable) {
 
     ac::video::DisplayOutput output;
     output.mode = ac::video::DisplayOutput::Mode::kExtend;
-    output.width = 640;
-    output.height = 480;
+    output.width = 1280;
```

Comment:
Does this hardcode resolution in any way? Whats the impact on the user

Reply:
Personally, I did not have a Miracast adapter that was capable of 1080p @ 60 Hz.
Theoretically the limit could be raised to 1080p, but that would probably require changes to system-settings.